roles/download/tasks/set_container_facts.yml

```diff
@@ -29,7 +29,7 @@
 
 - name: Set image save/load command for containerd
   set_fact:
-    image_save_command: "{{ containerd_bin_dir }}/ctr -n k8s.io image export {{ image_path_final }} {{ image_reponame }}"
+    image_save_command: "{{ containerd_bin_dir }}/ctr -n k8s.io image export --platform linux/{{ image_arch }} {{ image_path_final }} {{ image_reponame }}"
```

Comment:
I was wondering we would need to support Windows and this additional option could block it.
But Kubespray README.md clearly says Linux distributions are supported.
So it is fine to pass this option.

Reply:
At that point we would need to switch to `--all-platforms` instead if we wanted to support multi-platform or move to something like an iterative approach to downloading images where we iterate through all supported platforms and arch'es.

For now kubespray is Linux only so it should get the job done. I'm more concerned about the situation where we would want to support heterogeneous clusters with x86 and arm nodes. But our code needs further refinement for that as well.